README.md

```diff
@@ -1,10 +1,67 @@
-# rasmus_fuel
+# Fuel - based emission model for container ship
 
 [![Tests](https://github.com/willirath/rasmus_fuel/workflows/test/badge.svg)](https://github.com/willirath/rasmus_fuel/actions?query=workflow%3Atest)
 [![Coverage](https://codecov.io/gh/willirath/rasmus_fuel/branch/master/graphs/badge.svg?branch=master)](https://codecov.io/github/willirath/rasmus_fuel?branch=master)
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/willirath/rasmus_fuel/master?filepath=doc%2Fexamples)
 
-Fuel consumption paramerterisations for RASMUS.
+The model estimates at first step a fuel consumption and at seconds step a fuel-based CO_2 emission for a slow-speed (<400 rotation per minute (RPM)) container ships with a diesel engine 
+that runs on marine diesel oil (MDO). Total engine fuel consumption and emission are calculated for diesel engine type MAN-B&W with parameters described below. 
+
+### Model parameters
+
+In the model the convension of SI units for all parameters is implemented. Input parameters for the model are described in the table below. 
+
+| Parameter | Description , Units | 
+| --- | --- |
+| Vessel design speed | maxium speed vessel reach at calm conditions [m/s] |
+| Specific fuel consumption (SFOC)| specific fuel consumption, [kg/Wh]  |
+| Number of operational engines | Vessel number of operational engines |
+| Load per active engine | the torque output of an engine |
+| Maximum continuous rating (MCR)| maximum power output of engine|
+| Conversion factor of fuel to CO_2 | conversion factor from fuel to CO_2 mass [kg of fuel / kg CO_2]| 
+
+
+
+### Emission model description
+
+Depending on number of oepartional engines two alternative formulas for fuel consumption is implemented.
+To estimate a total fuel consumption with a single operational engine and auxiliary engine power the formulas is used [1]:
```

Comment:
For a single operational engine and auxiliary engine power, the total fuel consumption is estimated according to (see [1]):